### DISCUSSION TITLE
Modify repo file with Actions

### DISCUSSION THREAD

**Discussion by estensen** [community/community#25833] [Actions]
<p>I want to create an Action that changes a file in my repository. Files written to the local file system inside the Action will dissapear when the workflow is done. What is the best approach to modify files? Can Actions create commits?</p>

**Comment by itillis** [comment, marked as answer]
<p>Hi <a class="mention" href="/estensen">@estensen</a>,</p>
<p>With your GITHUB_TOKEN you can make changes as you describe. Here are a couple Actions from the community that might be a good place to start:</p>
<p><a href="https://github.com/cds-snc/github-actions/tree/master/auto-commit" rel="nofollow noopener">https://github.com/cds-snc/github-actions/tree/master/auto-commit</a></p>
<p><a href="https://github.com/ad-m/github-push-action" rel="nofollow noopener">https://github.com/ad-m/github-push-action</a></p>
<p>-Ilya</p>
